### DISCUSSION TITLE
New name?

### DISCUSSION THREAD

**Discussion by Ghoulboy78** [Ghoulboy78/World-edit-scarpet#47] [Ideas/Suggestions]
Suggestions for new names. Vote for them with 👍  or 👎

**Comment by Firigion** [comment]
My proposal is have a separate comment for each proposed name and anyone can vote on as many names as they want by reacting to the comments. We can have both a "full" official name and a a shorthand used as the base command. The comment proposing a name can also have a few short sentences stating why they think it's good.

To quote gnembon: "[...] the name of the app should be simple enough it rolls from command prompt. /we is good, cause on qwerty its just next to each other."

For example, for the current name:
**Full**: world-edit (or world-edit-scarpet?)
**Command**: /world_edit
**Features**: Clashes and creates confusion with the actual world-edit mod/plugin. It's also long to write when using commands.

Upvote this comment if you'd like to keep the current name. Use the up and downvotes on the top left, over there. This GitHub discussions thing is lit.

---

However, at the time of reposting this, all collaborators except for gnembon have voted already. Do we want to leave this open for a bit nevertheless, see if something happens? Maybe let non collaborators vote too?

@BisUmTo @altrisi @replaceitem if you don't mind voting again...

**Comment by Firigion** [comment]
**Full**: world-edit
**Command**: /we
**Features**: Easier to type; we don't need to change a bunch of thext in the app to accomodate for the new name, just suggested commands.

**Comment by Firigion** [comment]
**Full**: scarpet-edit
**Command**: /se
**Features**: Really easy to type too; play on the name of other world-edit type stuff (world edit, bling edit, prolly some others)

**Comment by Firigion** [comment]
**Full**: paint-app
**Command**: /pa
**Features**: Some may argue it's harder to type, because it's two hands, some may argue its easier, for the same reason. Name is a bit lame, tho.

**Comment by Firigion** [comment]
**Full**: worldoshop
**Command**: /ws
**Features**: idk, gnembon proposed it a while back. Command is still good in qwerty keyboards.
